third-party/linshare/src/test/java/org/apache/james/linshare/LinshareExtension.java

```diff
@@ -161,4 +165,8 @@ private void deleteAllUsersDocuments() {
         return Optional.ofNullable(USER_CREDENTIAL_MAP.get(user.getMail()))
             .orElseThrow(() -> new RuntimeException("cannot get token of user " + user.getMail()));
     }
+
+    public String getTechnicalAccountUUID() {
```

Comment:
what is it for?

Reply:
You can check it at [jira issue](https://issues.apache.org/jira/browse/JAMES-3040) and as i explained at https://github.com/linagora/james-project/pull/3090#discussion_r376205062